core/src/main/scalajvm/sttp/tapir/SchemaExtensions.scala

```diff
@@ -1,9 +1,11 @@
 package sttp.tapir
 
 import java.nio.file.Path
-
 import sttp.tapir.SchemaType.SBinary
 
+import java.io.File
+
 trait SchemaExtensions {
   implicit val schemaForPath: Schema[Path] = Schema(SBinary())
+  implicit val schemaForNormalFile: Schema[File] = Schema(SBinary())
```

Comment:
why not just `schemaForFile`? :)

Reply:
forFile was already taken by Schema[TapirFile]